unity-renderer/Assets/Desktop/Scripts/MainScripts/DCL/CommandLineParserDesktop/CommandLineParserUtils.cs

```diff
@@ -26,6 +27,9 @@ public static void ParseArguments()
                             i++; // shift
                             startPort = int.Parse(arguments[i]);
                             break;
+                        case "--without-ssl":
```

Comment:
Might I suggest

```suggestion
                        case "--no-ssl":
```
as an attempt to simplify the command line usage?